### ISSUE TITLE
How to get expression statistics?

### ISSUE THREAD

**Issue by soom1017** [gafniguy/4D-Facial-Avatars#32]
Thanks for previous support on [making continuous video](https://github.com/gafniguy/4D-Facial-Avatars/issues/31).

In the real_to_nerf.py code, it says I need "expressions.txt", "rigid.txt". 
Also in the json file of the person_1 dataset, there are "expressions" values that is already prepared.

How can I get these values from my own video or image sequence? I searched for face2face model code, and there's nothing but demo code using pix2pix model.

**Comment by yangqing-yq** [comment]
> You need a face tracker, you can try some open source ones, eg [](https://github.com/philgras/video-head-tracker)

as this vht repo "https://github.com/philgras/video-head-tracker", it outputs totally different dimension of expression vector (which is 100d), how to align this with nerface requirement (expression is 76D vector)?

**Comment by yangqing-yq** [comment]
@soom1017  hey, bro!
I am also stuck in the step of how to generate these "expressions.txt", "rigid.txt".
Have you finally figured it out?

**Comment by soom1017** [comment]
> @soom1017 hey, bro! I am also stuck in the step of how to generate these "expressions.txt", "rigid.txt". Have you finally figured it out?

Sorry for that.
As my team decided not to use nerf-like models, I have nothing proceeded.

**Comment by gafniguy** [comment]
@yangqing-yq yes, you can just change the 76 to the dimension of the FLAME expression vector.
With the rigid pose you have to be a bit more careful, as FLAME has a neck parameter as well, make sure you take that into account when you save the R|T of the head)
